test/configs/zombieMoonbeamRpc.json

```diff
@@ -48,7 +48,7 @@
       "collators": [
         {
           "name": "alith",
-          "command": "../target/release/moonbeam",
+          "command": "tmp/moonbeam_rt",
```

Comment:
@RomarQ we decided with Pablo to mvoe forward with this workaround, as we are confident that the problem doesn't exist in production. It's specific to *-local chain specs that use only for zombi tests. In a nutshell, it's because *-local chain spec use native runtime embedded in the client, so the new client create chain spec with state trie v1, that are not compatible with old runtime.
The proper fix should be to create json chain spec for zombi tests, but it's too much work for a temporary issue that we will have only for one release